addons/website_sale/models/product_template.py

```diff
@@ -729,6 +729,24 @@ def _get_images(self):
         self.ensure_one()
         return [self] + list(self.product_template_image_ids)
 
+    def _get_attrib_values_domain(self, attrib_values):
+        attrib = None
```

Comment:
```suggestion
        attribute_id = None
```

Maybe a good opportunity to clarify this code (and maybe rename some variable names?)
Found it a bit confusing when I read through it :D